docs/sources/reference/components/mimir.rules.kubernetes.md

```diff
@@ -24,6 +24,12 @@ in Kubernetes in order for {{< param "PRODUCT_NAME" >}} to access it via the Kub
 [Role-based access control (RBAC)]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
 {{< /admonition >}}
 
+{{< admonition type="warning" >}}
+This component does not support [clustered mode][] and must be run in a separate single-instance deployment of {{< param "PRODUCT_NAME" >}}.
```

Comment:
Do we want to include what happens if run in clustered mode anyway to drive home the issue?

Reply:
Good idea. Will update

Reply:
Fixed in c0cb0e473df81019879136363272e3d46e3b65ec